### DISCUSSION TITLE
Progress Bar

### DISCUSSION THREAD

**Discussion by litewrap** [AparokshaUI/Adwaita#16] [Q&A] [answered]
I would like to know if kind of Progress Bar similar to SwiftUI ProgressView https://developer.apple.com/documentation/swiftui/progressview is available with code example ?

**Comment by david-swift** [comment, marked as answer]
It is called [ProgressBar](https://aparokshaui.github.io/adwaita-swift/documentation/adwaita/progressbar).

An example implementation is available [here](https://github.com/david-swift/Memorize/blob/44609ea6aa96c4161340a7b612554166219dc739/Sources/View/StudyView.swift#L52). In the docs, in the methods section, you can find modifiers specific to this widget.

**Reply by litewrap** [reply]
Exactly what I was looking ! Thanks.
